src/condor_tests/test_issue_credentials.py

```diff
@@ -105,4 +118,21 @@ def test_top_file_created(self, the_test_name, the_condor, the_directory, the_us
             submit.issue_credentials()
 
         # Check for a .use file as well?
-        assert (Path(the_directory) / the_username / the_filename).exists()
+        assert (Path(the_directory) / the_username / the_local_filename).exists()
+
+    # Test that a user-added credential that's not managed by any credmons
+    # results in a .use file in the credd's storage.
+    def test_use_file_created(self, the_test_name, the_condor, the_directory, the_username, the_plain_filename, fake_user_token_file):
+        # Strictly speaking, this would be a set-up error.
+        assert not (Path(the_directory) / the_username / the_plain_filename).exists()
+
+        rv = the_condor.run_command(
+            ['htcondor', '-v', 'credential', 'add', 'oauth2',
+             str(fake_user_token_file),
+             '--service', 'plain'],
```

Comment:
Theoretically, the `'plain'` here should come from the test case; in practice, this can be left as-is until such a time as we decide we need to test that more than arbitrary token can be stored.

Reply:
Also: it's probably fine not to test that the "plain" credential can be used by a job successfully.